latex/Research_Strategy.tex

```diff
@@ -213,8 +219,8 @@
 %b.1 Figure showing progression in hydration free energy accuracy across a couple of SAMPLs or comparing two SAMPLs?
 %
  %Fennell SEA, etc.
-Shifts in protonation state and tautomer proved particularly important in the recent SAMPL5 $\log D$ challenge[refs], as they presumably will be for protein-ligand binding as well.
-Host-guest binding studies have also been particularly important[ref benchmark sets],
+Shifts in protonation state and tautomer proved particularly important in the recent SAMPL5 $\log D$ challenge [refs], which functions as a much more tractable venue for exploring the role these effects play in more complex protein-ligand binding.
```

Comment:
We need to communicate why it is useful to do physical property measurements that isolate specific physical effects from the more complex pharmacological target cases. I'm happy with any phrasing that communicates this clearly.